substrate/frame/im-online/Cargo.toml

```
@@ -17,17 +17,10 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { features = ["derive"], workspace = true }
-frame-benchmarking = { optional = true, workspace = true }
-frame-support = { workspace = true }
-frame-system = { workspace = true }
 log = { workspace = true }
-pallet-authorship = { workspace = true }
 scale-info = { features = ["derive", "serde"], workspace = true }
-sp-application-crypto = { features = ["serde"], workspace = true }
-sp-core = { features = ["serde"], workspace = true }
-sp-io = { workspace = true }
-sp-runtime = { features = ["serde"], workspace = true }
-sp-staking = { features = ["serde"], workspace = true }
+frame = { workspace = true, features = ["experimental", "runtime"] }
```

```suggestion
frame = { workspace = true, features = ["runtime"] }
```
Remove the `experimental` feature flag as in #7177